csharp/ql/src/semmle/code/csharp/ir/Util.qll

```diff
@@ -0,0 +1,47 @@
+/*
+ * Temporary file that has stubs for various functionalities in the IR conversion.
+ */
+
+import csharp
+
+class Locatable extends Element { }
```

Comment:
I'm not sure this is needed after my changes. You can just make `Language::AST` resolve to `Element`.